block-producer/src/batch_builder/batch.rs

```diff
@@ -27,9 +28,10 @@ pub struct TransactionBatch {
     id: BatchId,
     updated_accounts: BTreeMap<AccountId, AccountStates>,
     produced_nullifiers: Vec<Nullifier>,
-    created_notes_smt: BatchNoteTree,
     /// The notes stored `created_notes_smt`
```

Comment:
It seems this comment is not in the right place now. And I think, it is confusing a bit, maybe it would be better to get rid of this comment at all?

Reply:
Removed.